src/plugins/intel_gpu/src/graph/get_type_id.cpp

```diff
@@ -0,0 +1,198 @@
+#include "intel_gpu/primitives/extract_image_patches.hpp"
+#include "intel_gpu/primitives/eye.hpp"
+#include "intel_gpu/primitives/fully_connected.hpp"
+#include "intel_gpu/primitives/gather_elements.hpp"
+#include "intel_gpu/primitives/gather_nd.hpp"
+#include "intel_gpu/primitives/gather_tree.hpp"
+#include "intel_gpu/primitives/gather.hpp"
+#include "intel_gpu/primitives/gemm.hpp"
+#include "intel_gpu/primitives/generate_proposals.hpp"
+#include "intel_gpu/primitives/grn.hpp"
+#include "intel_gpu/primitives/grn.hpp"
+#include "intel_gpu/primitives/input_layout.hpp"
+#include "intel_gpu/primitives/loop.hpp"
+#include "intel_gpu/primitives/lrn.hpp"
+#include "intel_gpu/primitives/lstm_dynamic_input.hpp"
+#include "intel_gpu/primitives/lstm_dynamic_timeloop.hpp"
+#include "intel_gpu/primitives/lstm_dynamic.hpp"
+#include "intel_gpu/primitives/lstm.hpp"
+#include "intel_gpu/primitives/multiclass_nms.hpp"
+#include "intel_gpu/primitives/mutable_data.hpp"
+#include "intel_gpu/primitives/mvn.hpp"
+#include "intel_gpu/primitives/non_max_suppression.hpp"
+#include "intel_gpu/primitives/non_zero.hpp"
+#include "intel_gpu/primitives/normalize.hpp"
+#include "intel_gpu/primitives/one_hot.hpp"
+#include "intel_gpu/primitives/permute.hpp"
+#include "intel_gpu/primitives/pooling.hpp"
+#include "intel_gpu/primitives/prior_box.hpp"
+#include "intel_gpu/primitives/proposal.hpp"
+#include "intel_gpu/primitives/pyramid_roi_align.hpp"
+#include "intel_gpu/primitives/quantize.hpp"
+#include "intel_gpu/primitives/random_uniform.hpp"
+#include "intel_gpu/primitives/range.hpp"
+#include "intel_gpu/primitives/read_value.hpp"
+#include "intel_gpu/primitives/reduce.hpp"
+#include "intel_gpu/primitives/region_yolo.hpp"
+#include "intel_gpu/primitives/reorder.hpp"
+#include "intel_gpu/primitives/reorg_yolo.hpp"
+#include "intel_gpu/primitives/resample.hpp"
+#include "intel_gpu/primitives/reshape.hpp"
+#include "intel_gpu/primitives/reverse_sequence.hpp"
+#include "intel_gpu/primitives/reverse.hpp"
+#include "intel_gpu/primitives/roi_align.hpp"
+#include "intel_gpu/primitives/roi_pooling.hpp"
+#include "intel_gpu/primitives/roll.hpp"
+#include "intel_gpu/primitives/scatter_elements_update.hpp"
+#include "intel_gpu/primitives/scatter_nd_update.hpp"
+#include "intel_gpu/primitives/scatter_update.hpp"
+#include "intel_gpu/primitives/select.hpp"
+#include "intel_gpu/primitives/shape_of.hpp"
+#include "intel_gpu/primitives/shuffle_channels.hpp"
+#include "intel_gpu/primitives/slice.hpp"
+#include "intel_gpu/primitives/softmax.hpp"
+#include "intel_gpu/primitives/space_to_batch.hpp"
+#include "intel_gpu/primitives/space_to_depth.hpp"
+#include "intel_gpu/primitives/split.hpp"
+#include "intel_gpu/primitives/strided_slice.hpp"
+#include "intel_gpu/primitives/tile.hpp"
+
+cldnn::primitive_type_id cldnn::get_type_id(std::string type_str) {
+    static std::unordered_map<std::string, primitive_type_id> primitive_inst_types = {
+        {"activation", cldnn::activation::type_id()},
```

Comment:
2 new lists of primitives in single PR is too much...
Please come up with a better solution

Reply:
I also removed it. Thank you.